nautobot/utilities/tests/test_forms.py

```diff
@@ -429,3 +431,52 @@ def test_invalid_input(self):
         for test in tests:
             with self.assertRaises(forms.ValidationError):
                 self.field.clean(test)
+
+
+class AddressFieldMixinTest(TestCase):
+    """Test cases for the AddressFieldMixin."""
+
+    def setUp(self):
+        """Setting up shared variables for the AddressFieldMixin."""
+        self.dummy_address = "10.0.0.1/24"
+        self.initial = {"address": self.dummy_address}
+
+    def test_address_initial(self):
+        """Ensure initial kwargs for address is passed in."""
+        with mock.patch("nautobot.utilities.forms.forms.forms.ModelForm.__init__") as mock_init:
+            AddressFieldMixin(initial=self.initial, instance={})
+            mock_init.assert_called_with(initial=self.initial, instance=mock.ANY)
+
+    def test_address_instance(self):
+        """Ensure override with computed field when initial kwargs for address is not passed in."""
+
+        class DummyInstance:
+            address = self.dummy_address
+
+        with mock.patch("nautobot.utilities.forms.forms.forms.ModelForm.__init__") as mock_init:
+            AddressFieldMixin(instance=DummyInstance())
+            mock_init.assert_called_with(initial=self.initial, instance=mock.ANY)
```

Comment:
Consider using an actual IPAddress instance?

```suggestion
        ip_address = IPAddress(address=self.dummy_address)

        with mock.patch("nautobot.utilities.forms.forms.forms.ModelForm.__init__") as mock_init:
            AddressFieldMixin(instance=ip_address)
            mock_init.assert_called_with(initial=self.initial, instance=ip_address)
```

(Of course you'll need `from nautobot.ipam.models import IPAddress` as well.)

Similar suggestion below for using a `Prefix`.

Reply:
I could... the point of the DummyInstance is that we aren't testing the instance functionality so much as just testing the AddressFieldMixin...

Reply:
Yeah, may be a matter of personal preference. :-) IMHO using an actual instance object helps us guarantee that the "real" code path is what gets exercised by the test, but it's not critical.